site/src/main/java/uk/nhs/digital/indices/StickySection.java

```diff
@@ -0,0 +1,28 @@
+package uk.nhs.digital.indices;
+
+public interface StickySection {
+
+    /**
+     * The StickySection's Title.
+     *
+     * @return A String that can be empty.
+     */
+    String getHeading();
+
+    /**
+     * The type of heading.
+     *
+     * @return The name of the heading level.
+     */
+    String getHeadingLevel();
```

Comment:
Could we also have default implementation of getHeading() and getHeadingLevel() - similar as implementation in Download.java? 

Thanks to that we will not need to dupliate the actual implementation in classes as Download, GallerySection etc. and similar future classes.

Reply:
I did think about this. But I deiced there was no clear default.

The best default would be `return getProperty("website:headinglevel");`. However this make too much of an assumption on the doc type.

Let me know what you think?

Reply:
We can leave it as it is for now. I see the 2 classes (Download, GallerySection) where actual default could work. If we will have more classes in the future we could refactor this part then.